src/server/standard_address_space_part8.cpp

```diff
@@ -698,7 +698,8 @@ namespace OpcUa
         attrs.Type = ObjectId::String;
         attrs.Rank = 1;
         attrs.IsAbstract = false;
-        attrs.Dimensions = std::vector<uint32_t>({0});
+        attrs.Dimensions = std::vector<uint32_t>();
+		attrs.Dimensions.push_back(0);
```

Comment:
standard_address_space_partXXXX are autogenerated files, you should not modify them. look at the script schemas/generate_address_space.py and the method called to_vector. there you can modify how the vector init code is generated